inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/max_pool.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
There is max_pooling in `pooling.cpp`. Please, do not duplicate. review it (pooling.cpp)